cfme/utils/appliance/__init__.py

```diff
@@ -468,6 +474,78 @@ def configure(self, log_callback=None, **kwargs):
                 self.restart_evm_service(log_callback=log_callback)
             self.wait_for_web_ui(timeout=1800, log_callback=log_callback)
 
+    def configure_rhos_db_disk(self):
+        loopback_script_path = "/usr/local/sbin/loopbacks"
+        loopback_script_content = """EOF
```

Comment:
**REQUIRED**: Please use `textwrap.dedent` on these strings just to be on the safe side with whitespace :)

```python
from textwrap import dedent

        loopback_script_content = dedent("""EOF
        #!/bin/bash
        ...
        EOF"""
```